variables.tf

```diff
@@ -1,5 +1,44 @@
-#variable "example" {
-#  type        = string
-#  default     = ""
-#  description = "This is just an example of a variable"
-#}
+# Required
+
+variable "tenancy_id" {
+  type        = string
+  description = "OCID of the OCI tenancy to be integrated with Lacework"
+}
+
+# Optional
+
+variable "create" {
+  type        = bool
+  default     = true
+  description = "Set to false to prevent module from creating any resources"
+}
+
+variable "freeform_tags" {
+  type        = map(any)
+  default     = {}
+  description = "freeform tags for the resources created for Lacework integration"
+}
+
+variable "email" {
+  type        = string
+  default     = "lacework@lacework.net"
```

Comment:
Maybe we should have no default for email since this won't be monitored?

Reply:
Good idea. For some reason, I thought the email was a required parameter, but I see now that it's an optional parameter. Will remove it.

Reply:
Turns out it is a required parameter. The Terraform docs say it's optional, but the OCI API returns an error (""The primary email must be specified") if you don't specify an email. Leaving it as an empty string the same error.